praw/models/reddit/removal_reasons.py

```diff
@@ -80,21 +80,21 @@ def __init__(
             compatibility. This parameter should not be used.
 
         """
-        reason_id = self._warn_reason_id(id_value=id, reason_id_value=reason_id)
-        if (reason_id, _data).count(None) != 1:
+        _reason_id = self._warn_reason_id(id_value=id, reason_id_value=reason_id)
+        if (_reason_id, _data).count(None) != 1:
             msg = "Either id or _data needs to be given."
             raise ValueError(msg)
 
-        if reason_id:
-            self.id = reason_id
+        if _reason_id:
```

Comment:
Since you are using this value, it seems a little strange to `_` prefix it.

Reply:
I think a linter was whining about it possibly mutating `reason_id`. Looking at `self._warn_reason_id`, that shouldn't be an issue.